### DISCUSSION TITLE
Large numbers of "session restored from cache has expired" warnings.

### DISCUSSION THREAD

**Discussion by bitmagewb** [OpenIDC/mod_auth_openidc#794] [Q&A] [answered]
I've protected a site and things seem to be working as expected.  Users are prompted to login, and get access to protected content once they have done so.  But the server log has a flood of warnings:
[Thu Feb 17 16:42:25.889635 2022] [auth_openidc:warn] [pid 25144] [client 71.167.203.157:0] oidc_session_extract: session restored from cache has expired, referer: https://-protected page on site-

Are those generated for unauthenticated accesses, or do they represent another issue?  The OIDC provider is Okta.

**Comment by zandbelt** [comment, marked as answer]
these warnings are not about unauthenticated access, they are about expired sessions, unrelated to the Provider, usually after a period of user inactivity (defaulting to 5 mins)

**Reply by bitmagewb** [reply]
Thank you.  Is that controlled by the OIDCSessionInactivityTimeout setting?

**Reply by zandbelt** [reply]
yes

**Reply by bitmagewb** [reply]
Thanks!
